lib/stuff.php

```diff
@@ -247,6 +263,8 @@ function launch_wordpress( $php_version = 'default', $requested_features = array
 			debug( '%s: Stopping pings to Ping-O-Mattic', $app->domain );
 			stop_pingomatic();
 
+			add_other_users();
```

Comment:
Looks like everything else here has a debug statement. The statements seem kind of pointless since these functions all just add filters that run later, but 🤷
```suggestion
			debug( '%s: Adding additional users', $app->domain );
			add_other_users();
```